src/components/QuranReader/QuranReader.module.scss

```diff
@@ -30,3 +31,15 @@
   display: flex;
   justify-content: center;
 }
+
+.loadMoreContainer {
+  text-align: center;
+  button {
+    background: none !important;
```

Comment:
strongly advise against using `!important`

We can extend the button component with props that give us background none, etc.